core/src/test/java/uk/gov/gchq/koryphe/signature/SignatureTest.java

```diff
@@ -379,7 +379,7 @@ public String apply(final Long l, final Double d) {
     @Test
     public void shouldGenerateInputSignatureUsingTupleClassWhenNotParameterised() {
         // Given
-        final Function<TestTuple, String> fromTestTuple = new KorypheFunction<TestTuple, String>() {
+        final Function<TestTuple, String> fromTestTuple = new KorypheFunction<>() {
```

Comment:
Was this required or just a clean up? Is it better to use nothing here?

Reply:
Its not required by the IDE/Compiler visually cleaner.

Reply:
Isn't it better to declare types though? Or are these types already enforced and writing them again is too verbose?

Reply:
Okay the types are also spelled out as well on the left hand side so this is really just cleaning things up, noted